backend/src/transaction_event/protocol_level_tokens.rs

```diff
@@ -720,9 +723,9 @@ impl PreparedTokenUpdate {
                         )
                         SELECT 
                             $1,
-                            lm.prev_transfer_amount + $2,
+                            lm.prev_transfer_amount + ($2 / POWER(10::numeric, ti.decimal)),
```

Comment:
I would do/keep math outside the `QUERY` as much as possible as the `QUERY` becomes hard to read and likely becomes less performant. Since amounts are normalized in the code base in other places, do we have a helper function for it already?